addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -313,6 +314,14 @@ class Composer extends Component {
      * @private
      */
     _onInputTextInput() {
+        this.storeDispatch(
+            'saveComposerContent',
+            this.props.composerLocalId,
+            this._textInputRef.comp.getContent(),
+            this._textInputRef.comp.getSelectionStart(),
+            this._textInputRef.comp.getSelectionEnd(),
+            this._textInputRef.comp.getHeight(),
+        );
         this.state.hasTextInputContent = !this._textInputRef.comp.isEmpty();
```

Comment:
Not sure if this variable is still used, but it should now be possible to compute if from the content in the store directly.

Reply:
I removed it and replaced it by `storeProps.composer.textInputContent` where it was used in the composer view